### PR TITLE
Dbatiste/update deps

### PR DESCRIPTION
**Pull request by dbatiste**
Key dep updates - d2l-icons, d2l-dropdown (refer rendering in DOM until opened).

### CODE REVIEW

**Comment by awikkerink** [comment]
`Unable to find suitable version for d2l-offscreen`

**Comment by awikkerink** [comment]
Might be a cache issue. Cleared caches and restarted build

**Comment by awikkerink** [comment]
Seems to be working now

**Comment by awikkerink** [comment]
@dbatiste could you merge https://github.com/BrightspaceUI/button-group/pull/32 onto this one? It should fix the build

File: d2l-buttons-menu.html [hunk 1]
```diff
@@ -158,6 +158,9 @@
 
 				// create menu items from template items
 				var items = Polymer.dom(this._instance.root).childNodes;
+
+				this.$$('d2l-dropdown-menu').forceRender();
```
Comment (dbatiste) [inline d2l-buttons-menu.html]: This `forceRender` method will cause synchronous render of the internal `template` that's normally used to defer the rendering of the dropdown contents until first-open.

Comment (awikkerink) [inline d2l-buttons-menu.html]: That would only work if there is one d2l-dropdown-menu. It should be moved to `_createMenuItem`

Comment (awikkerink) [inline d2l-buttons-menu.html]: I like this approach better

**Comment by dbatiste** [comment]
Thanks for the help with this Anthony.  :)

**Pull request merged**